compiler/src/middle_end/matchcomp.re

```diff
@@ -616,19 +616,19 @@ let rec default_matrix = mtx => {
   );
 };
 
-let rec compile_matrix = (branches, mtx) =>
+let rec compile_matrix = mtx =>
```

Comment:
I cleaned this up so we don't have to pass the branches through. That was originally necessary because we assigned branch numbers along the way, but some time ago we switched to assigning each branch its own number before compiling.

Reply:
Otherwise, this function hasn't changed.

Reply:
🙏